include/z64cutscene_commands.h

```diff
@@ -11,445 +11,273 @@
-
-/**
- * ARGS
- *   s8 continueFlag (c), s8 roll (r), s16 frame (f), f32 viewAngle (a),
- *   s16 xPos (x), s16 yPos (y), s16 zPos (z)
- * FORMAT
- *   Capital U is Unused
- *   ccrrffff aaaaaaaa xxxxyyyy zzzzUUUU
- *   size = 0x10
- */
-#define CS_CAM_POS CS_CAM_EYE
-#define CS_CAM_EYE(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
-    CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
-
-/**
- * ARGS
- *   s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   00000002 0001ssss eeee0000
- *   size = 0xC
- */
-#define CS_CAM_FOCUS_POINT_LIST CS_CAM_AT_LIST
-#define CS_CAM_AT_LIST(startFrame, endFrame) \
-    CS_CMD_CAM_AT, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
-
-/**
- * ARGS
- *   s8 continueFlag (c), s8 roll (r), s16 frame (f), f32 viewAngle (a),
- *   s16 xPos (x), s16 yPos (y), s16 zPos (z)
- * FORMAT
- *   Capital U is Unused
- *   ccrrffff aaaaaaaa xxxxyyyy zzzzUUUU
- *   size = 0x10
- */
-#define CS_CAM_FOCUS_POINT CS_CAM_AT
-#define CS_CAM_AT(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
+#define CS_CAM_POINT(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
     CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   00000003 eeeeeeee
- *   size = 0x8
+ * Declares a list of camera `eye` points that form a spline.
+ * The points of the spline are interpolated over time to create smooth camera movements.
  */
-#define CS_MISC_LIST(entries) CS_CMD_MISC, CMD_W(entries)
+#define CS_CAM_EYE_SPLINE(startFrame, endFrame) \
+    CS_CMD_CAM_EYE_SPLINE, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
 
 /**
- * ARGS
- *   s16 unk (u), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   uuuussss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU
- *   size = 0x30
+ * Declares a list of camera `at` points that form a spline.
+ * The points of the spline are interpolated over time to create smooth camera movements.
  */
-#define CS_MISC(unk, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7, unused8, unused9, unused10) \
-    CMD_HH(unk, startFrame), CMD_HH(endFrame, unused0), \
-    CMD_W(unused1), CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), \
-    CMD_W(unused6), CMD_W(unused7), CMD_W(unused8), CMD_W(unused9), CMD_W(unused10)
+#define CS_CAM_AT_SPLINE(startFrame, endFrame) \
+    CS_CMD_CAM_AT_SPLINE, CMD_HH(0x0001, startFrame), CMD_HH(endFrame, 0x0000)
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   00000004 eeeeeeee
- *   size = 0x8
+ * Declares a list of camera `eye` points that form a spline.
+ * The points of the spline are interpolated over time to create smooth camera movements.
+ * The camera is moved relative to player.
```

Comment:
it's probably not important to go into details but to be specific, the positions are translated relative to the player, and rotated around the player by the player's (world) yaw

if "move" could make it seem like there's only a translation going on and no rotation, maybe:
```suggestion
 * The points are relative to the player.
```

relevant z_camera code:
https://github.com/zeldaret/oot/blob/aa48c66e10fc067e4d41c3a1d097bd83dea53acf/src/code/z_camera.c#L5766-L5775

Reply:
Or:
```suggestion
 * The points are relative to the player's position and yaw.
```
```suggestion
 * The points are made relative to the player's position and yaw.
```